docs/src/main/sphinx/release/release-376.md

```diff
@@ -18,7 +18,8 @@
 
 ## Security
 
-* Add support to use two-way TLS/SSL certificate validation with LDAP authentication. ({issue}`11070`)
+* Add support to use two-way TLS/SSL certificate validation with LDAP authentication. ({issue}`11070`).
+* Replace `ldap.ssl-trust-certificate` config with `ldap.ssl.truststore.path`. ({issue}`11070`). 
```

Comment:
This should NOT be a separate bullet item but just a separate sentence in the same item